sass/cd/cd-header/_cd-header.scss

```diff
@@ -15,8 +15,8 @@
   position: relative;
   height: var(--cd-global-header-height);
   color: var(--cd-white);
-  background: var(--cd-primary);
-  background-image: linear-gradient(-180deg, var(--cd-primary) 57%, var(--cd-primary--dark) 97%);
+  background: var(--brand-primary);
+  background-image: linear-gradient(-180deg, var(--brand-primary) 67%, var(--brand-primary--dark) 97%);
```

Comment:
It has been said that the gradient makes the design look quite dated at this point. I don't think anyone would be sad if this went away :)